test/e2e/storage/testsuites/volumemode.go

```diff
@@ -481,32 +480,3 @@ func listPodDirectory(h utils.HostExec, path string, node *v1.Node) ([]string, e
 	}
 	return strings.Split(out, "\n"), nil
 }
-
-// NOTE(avalluri): The below code is intentionally copied from e2e/common package.
```

Comment:
Does this still apply?

Reply:
No, all of the copied code gets removed by this PR.